app/assets/stylesheets/components/_contact_panels.scss

```diff
@@ -0,0 +1,90 @@
+.l-contact-panel {
+  @include column(4);
+}
+
+.contact-panel {
+  @include column(12);
```

Comment:
Shouldn't need this here: it adds extra horizontal margins to the contact panels.

Reply:
This has been removed